src/Integration.Vsix/SonarLint.vsct

```diff
@@ -28,8 +28,13 @@
       <Group guid="SonarLintCmdSet" id="SonarLintOuterGroup" priority="0x0100">
         <Parent guid="guidSHLMainMenu" id="IDM_VS_MENU_EXTENSIONS" />
       </Group>
-      <!-- The sub menu that parents commands. -->
-      <Group guid="SonarLintCmdSet" id="SonarLintGroup" priority="0x0100">
+      <Group guid="SonarLintCmdSet" id="SonarLintConnectedModeGroup" priority="0x0100">
+        <Parent guid="SonarLintCmdSet" id="SonarLintMenu" />
+      </Group>
+      <Group guid="SonarLintCmdSet" id="SonarLintCommandsGroup" priority="0x0101">
```

Comment:
I renamed SonarLintGroup to SonarLintCommandsGroup and created 2 additional groups for help and connected mode; this is so I could have the dividers.

Reply:
Nice!